php_dbus.h

```diff
@@ -27,7 +27,138 @@
 
 #include "Zend/zend_hash.h"
 
-#define PHP_DBUS_VERSION "0.1.0"
+#define PHP_DBUS_VERSION "0.1.2"
```

Comment:
I think it would be better to make this at least ``0.2.0``?

Reply:
done.